src/identity/hooks/useAttestAddress.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { base } from 'viem/chains';
+import type { Address, Chain } from 'viem';
+
+import { getEASAttestations } from '../getEASAttestations';
+import { useIdentity } from './useIdentity';
+import { attestationMapping } from '../attestationMapping';
+
+type UseAttestAddress = {
+  address: Address;
+  chain?: Chain;
+};
+
```

Comment:
@kyhyco what's our goal with `useAttestAddress`, can you share it out loud please. I am trying to think on a better name and better return type.

Reply:
`useAttestAddress` will return `null` if user is not attested. Return `eas` or `nouns` (whatever attests the user) if attested.

Reply:
changed to `useAttestation`